index.js

```diff
@@ -0,0 +1,48 @@
+// When a block is clicked, it should go to the top of the stack
+// ROCKETS
+const blocks = document.querySelector('.blocks')
+const childrenArr = [...blocks.childNodes]// spread syntax, not an array 
+
+const toTop = (event) => {
+  let childToMove = blocks.removeChild(event.target)
+  blocks.insertBefore(childToMove,blocks.firstChild)
+  event.stopPropagation()
+}
+
+// add event listener to each child
+childrenArr.map(child => {
+  child.addEventListener('click', toTop)
+})
+
+let interval
+let mouse = false
+// While a mouse is clicked down on a box, it should move to the right
+// TRAVELERS
+
+const moveToRight = node => {
```

Comment:
Take a closer look at this function - when you open your html, you will notice that your boxes move to the left rather than to the right. Off the top of your head, what is something that you can change??